lib/fpm/package/rpm.rb

```diff
@@ -512,6 +515,14 @@ def build_sub_dir
     #return File.join("BUILD", prefix)
   end # def build_sub_dir
 
+  def summary
+    if !attributes[:rpm_summary]
+      return @description.split("\n").first.empty? ? "_" : @description.split("\n").first
```

Comment:
See #973. @description could be an empty string and this will crash:

```
>> "".split("\n").first
=> nil
```

The crash will be `NoMethodError: undefined method `empty?' for nil:NilClass`